module-sys/PhoneModes/Observer.cpp

```diff
@@ -7,25 +7,6 @@
 
 namespace sys::phone_modes
 {
-    namespace
-    {
-        template <typename F> void onChanged(const F &onChanged, const Observer::OnFinishedCallbacks &onFinished)
```

Comment:
Just to make sure: this observer is only used in the changed functionality and does not impact others?

Reply:
this was useless code - it did literally nothing at the moment of removal. 

saying that - with this code removed we can clearly see that we have one observer which has to be modified each time we want to observe something different. We should have separate, can be templated, observers per thing we want to observe.
This 3 min change should explain what I've in mind: 
![image](https://user-images.githubusercontent.com/6411862/154941243-cb42d81b-20df-4358-8d7f-3c3ec5b73396.png)